### PR TITLE
Design Improvements no. 5

### PR DESCRIPTION
**Pull request by D0nPiano**
### Feedbacklist:
- amount of saved food for stores in message above

### Layout
- removed right whitespace again (off-centered login boxes before etc.)
  
### Feedback List - @tiltec 's requests in #639 
- [x] pickupFeedback form: highlight "other pickups" button more
- [x] amountviewer: make number less jumpy when sliding, e.g. by always showing the decimal
- [x] Rename 'you took home' -> 'how much food was saved on this pickup' based on feedback from Bruno in https://github.com/yunity/karrot-frontend/issues/159#issuecomment-328352706
- [x] Show group sidenav when in feedback page
- [x] feedbackList: show hint when there is no feedback yet
- [x] FeedbackItem: linkify user name and store name
- [x] use editMixin for pickupFeedback form to also allow editing feedback (see isEditable attribute of feedback to check if editing is allowed)
- [x] Add infinite scroll to feedback list to have proper pagination support
- [x] Remove feedback tab in pickups, add feedback to sidebar instead; I think we shouldn't introduce yet another navigation concept

### Stuff for some other PR
- [ ] Show FeedbackNotice in group/store feedback lists (maybe needs another component layer)
- [ ] pickupFeedback form: show pending status on create button
- [ ] add toast after you created feedback saying "Thank you <q-btn give more feedback/>
- [ ] update available-feedback after user created feedback (currently, x does not change in 'you can give feedback to x pickups' after you gave feedback)

### CODE REVIEW

**Comment by tiltec** [comment]
> use editMixin for pickupFeedback form to also allow editing feedback (see isEditable attribute of feedback to check if editing is allowed)
> Add infinite scroll to feedback list to have proper pagination support

I'll add these in another PR.

> perhaps change default ordering of feedback to sort by pickup date, not by feedback creation date

I would change this directly in the backend if we agree on it.

File: src/components/Pickups/PickupItem.vue [hunk 1]
```diff
@@ -10,7 +10,7 @@
           <slot>Date or Store Slot</slot>
         </div>
         <div
-          class="description multiline"
+          class="description"
```
Comment (tiltec) [inline src/components/Pickups/PickupItem.vue]: Why did you change this? The pickup comment is supposed to support line breaks, as the input field is a textarea.

File: src/pages/Store/Feedbacks.vue [hunk 2]
```diff
@@ -1,10 +1,65 @@
...
+  components: { FeedbackList, QCard, QTooltip, QBtn },
+  computed: {
+    ...mapGetters({
+      feedback: 'feedback/filtered',
+      fetchStatus: 'feedback/fetchStatus',
+    }),
+    totalAmount () {
+      let amount = 0
+      for (let fb in this.feedback) {
+        console.log(this.feedback[fb])
```
Comment (tiltec) [inline src/pages/Store/Feedbacks.vue]: Remove console.log ;)

File: src/components/Statistics/FeedbackItem.vue [hunk 3]
```diff
@@ -72,9 +80,16 @@ export default {
...
     },
+    storeId () {
+      const { about: { store: { id } = {} } = {} } = this.feedback
+      return id
+    },
     userName () {
       return this.feedback && this.feedback.givenBy && this.feedback.givenBy.displayName
     },
+    userId () {
+      return this.feedback && this.feedback.givenBy && this.feedback.givenBy.id
```
Comment (tiltec) [inline src/components/Statistics/FeedbackItem.vue]: I think we need a nicer way to resolve properties without throwing errors, maybe I'll add a helper soon...

File: src/components/Pickups/PickupItem.vue (hunk 1 above)
Comment (D0nPiano) [reply src/components/Pickups/PickupItem.vue]: Multiline adds a linebreak at the very beginning, making it look like this:
multiline-on

Without it, it still supports line breaks, and looks like this:
multiline-off

File: src/pages/Store/Feedbacks.vue (hunk 2 above)
Comment (D0nPiano) [reply src/pages/Store/Feedbacks.vue]: Oups :laughing:

**Comment by D0nPiano** [comment]
What about:
feedbackitem

I think its better than before, but I'm still not perfectly sure about the header (Store & date) - definitely open for more ideas :D

**Comment by D0nPiano** [comment]
What about:

feedbackitem2


I think its better than before, but I'm still not perfectly sure about the header (Store & date) - definitely open for more ideas :D

File: src/components/Pickups/PickupItem.vue (hunk 1 above)
Comment (tiltec) [reply src/components/Pickups/PickupItem.vue]: I mean intentional linebreaks, when somebody hits 'enter' to create a newline.

Comment (tiltec) [reply src/components/Pickups/PickupItem.vue]: I entered some linebreaks. It should show like this:

image

With this PR, it shows like this:

image

**Comment by tiltec** [comment]
The new design looks good! I would remove the comma after the date though.. similar to the group pickup view:

image

File: src/components/Pickups/PickupItem.vue (hunk 1 above)
Comment (tiltec) [reply src/components/Pickups/PickupItem.vue]: The preceding space is annoying though, I wonder where it comes from...

Comment (D0nPiano) [reply src/components/Pickups/PickupItem.vue]: Haha, I found it! :laughing: it was in the HTML - see my last commit

File: src/components/Statistics/FeedbackItem.vue [hunk 4]
```diff
@@ -28,8 +28,12 @@
               <DateAsWords :date="createdAt" />
             </span>
           </small>
-          <div class="comment">{{ comment }}</div>
-          <div>
+          <div
+            v-if="comment"
+            class="comment">
```
Comment (tiltec) [inline src/components/Statistics/FeedbackItem.vue]: Ah, I just saw that the comment input is a textarea. So it would make sense to make it `multiline` too.

Comment (D0nPiano) [reply src/components/Statistics/FeedbackItem.vue]: Yeah, makes sense. Was also just wondering about that. -> Changed it :white_check_mark:
